components/layout/webrender_helpers.rs

```diff
@@ -298,9 +298,7 @@ impl WebRenderStackingContextConverter for StackingContext {
                                 mut scroll_policy: ScrollPolicy,
```

Comment:
Yeah, that's true. I already removed the variable in the Servo half of https://github.com/servo/webrender/pull/451, so I didn't redo it here. It was sort of an accident that this bit of code was also affected by this PR.